planet/Objects.ocd/Environment.ocd/Stalactite.ocd/Script.c

```diff
@@ -21,7 +21,7 @@ public func Construction()
 		sibling = TransformBone(bone, transformation, 1, Anim_Const(1000), sibling);
 	}
 	
-	AddTimer("DoWaterdrop", RandomX(36 * 10, 36 * 40));
+	AddRainDropEffect(nil, 100, "Water", 2);
```

Comment:
Because the raindrop interval is not randomized, all stalactites in the game will drop at the same time (and also play the sound at the same time), which might be veeeery weird ingame. Also you drop & sound around nine times more often than before (on average)